.github/workflows/release.yaml

```diff
@@ -0,0 +1,135 @@
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lowercase IMAGE_PATH
+      id: image_path_format
+      uses: ASzc/change-string-case-action@v2
+      with:
+        string: ${{ env.IMAGE_PATH }}
+
+    - name: Build and push img with latest tag
+      id: explorer_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/explorer:latest
+    
+    - name: Build and push img with latest tag
+      id: explorer-db_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./postgres-Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/explorer-db:latest
+
+
+  # on tag vx.y.z created, create docker tag x.y.z
+  build-and-push-docker-release:
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set output
+      id: vars
+      run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/v}
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lowercase IMAGE_PATH
+      id: image_path_format
+      uses: ASzc/change-string-case-action@v2
+      with:
+        string: ${{ env.IMAGE_PATH }}
+
+    - name: Build and push release tag based on release ${{ steps.var.outputs.tag }}
+      id: explorer_db_release
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./postgres-Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/explorer-db:${{ steps.vars.outputs.tag }}
+    
+    - name: Build and push release tag based on release ${{ steps.var.outputs.tag }}
+      id: explorer_release
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/explorer:${{ steps.vars.outputs.tag }}
```

Comment:
Q: Can you please check the auto-version increment step as well?

Reply:
version is based on release tag, anytime a release tag is created, the same will be used as version.

Reply:
Please dont merge this PR as there is a dependency on of the test-case(failing).